docs/gatsbyjs.md

```diff
@@ -73,3 +73,19 @@ npm run develop
 
 This will run the Gatsby app with hot-reloading on `http://localhost:8000` and
 use `https://dev.telescope.cdot.systems` as your backend api.
+
+Please note `npm run develop` will detect 99% of any issues that may occur when working on the front-end. It is highly advised to run `npm run build` to verify there are no issues.
+
+```
+npm run build
+```
+
+If the build has been completed without any issues, it can now be served using:
+
+```
+npm run serve
```

Comment:
You only need this if you are not using `develop`.  I would probably drop the `build` and `serve` bits.